backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/index/impl/ElasticsearchIndexManagerBuilder.java

```diff
@@ -71,27 +76,31 @@ public ElasticsearchIndexManagerImpl build() {
 
 		ElasticsearchWorkOrchestrator parallelOrchestrator = null;
 		ElasticsearchWorkOrchestrator serialOrchestrator = null;
+		ElasticsearchIndexManagerImpl indexManager = null;
 
 		try {
 			parallelOrchestrator = indexingBackendContext.createParallelOrchestrator( elasticsearchIndexName );
 			serialOrchestrator = indexingBackendContext.createSerialOrchestrator( elasticsearchIndexName, refreshAfterWrite );
 
-			// TODO make sure index initialization is performed in parallel for all indexes?
-			indexingBackendContext.initializeIndex( parallelOrchestrator, encodedElasticsearchIndexName, encodedTypeName, model )
-					.join();
-
-			return new ElasticsearchIndexManagerImpl(
+			indexManager = new ElasticsearchIndexManagerImpl(
 					indexingBackendContext, searchBackendContext,
 					hibernateSearchIndexName, encodedElasticsearchIndexName,
-					encodedTypeName,model,
+					encodedTypeName, model,
+					indexManagementStrategy,
 					serialOrchestrator, parallelOrchestrator,
 					refreshAfterWrite
 			);
+
+			// TODO perform index initialization in parallel for all indexes?
```

Comment:
I think that `indexManager.start()` could be moved outside from the builder.
One option could be to move it to the constructor of `ElasticsearchIndexManagerImpl`, another option could be to move it to the builder client class.